### PR TITLE
Update to integrate JOSS review from @arbennett

### PR DESCRIPTION
**Pull request by ebocher**
This PR intregates some changes : 

1. Replace 2.5 geometries by 2D geometries. Wall and roof values are attributes to the spatial feature.
2. Clarify Block scale definition
2. Clarify RSU definition

### CODE REVIEW

File: docs/articles/joss/paper.md [hunk 1]
```diff
@@ -81,9 +81,9 @@ Table 1: GeoClimate features description and differences with similar commonly-u
 
 GeoClimate performs indicators computation at three spatial unit scales, a spatial unit being a POLYGON or MULTIPOLYGON geometry:
 
-1. Building scale, defined as a collection of features represented by 2.5 geometries with attributes as measures for walls and roof, 
-2. Block scale, defined as a set of buildings touching each other (at least one point in common) or as an isolated building, 
-3. Reference Spatial Unit (RSU) scale, being the elementary unit to characterize all the characteristics of a piece of land (not only related to buildings but also to vegetation, water). 
+1. Building scale, defined as a collection of features represented by 2D geometries with attributes as measures for walls and roof.
```
Comment (j3r3m1) [inline docs/articles/joss/paper.md:84]: keep between parenthesis 2.5D at the end of this sentence ?

File: docs/articles/joss/paper.md [hunk 2]
```diff
@@ -81,9 +81,9 @@ Table 1: GeoClimate features description and differences with similar commonly-u
 
 GeoClimate performs indicators computation at three spatial unit scales, a spatial unit being a POLYGON or MULTIPOLYGON geometry:
 
-1. Building scale, defined as a collection of features represented by 2.5 geometries with attributes as measures for walls and roof, 
-2. Block scale, defined as a set of buildings touching each other (at least one point in common) or as an isolated building, 
-3. Reference Spatial Unit (RSU) scale, being the elementary unit to characterize all the characteristics of a piece of land (not only related to buildings but also to vegetation, water). 
+1. Building scale, defined as a collection of features represented by 2D geometries with attributes as measures for walls and roof. 
+2. Block scale, defined as the union of the geometries of buildings touching each other (at least one point in common). Note that the union can represent a single building.
```
Comment (j3r3m1) [inline docs/articles/joss/paper.md:85]: Note that an isolated building is then also consider as a block ?

File: docs/articles/joss/paper.md [hunk 3]
```diff
@@ -81,9 +81,9 @@ Table 1: GeoClimate features description and differences with similar commonly-u
...
-2. Block scale, defined as a set of buildings touching each other (at least one point in common) or as an isolated building, 
-3. Reference Spatial Unit (RSU) scale, being the elementary unit to characterize all the characteristics of a piece of land (not only related to buildings but also to vegetation, water). 
+1. Building scale, defined as a collection of features represented by 2D geometries with attributes as measures for walls and roof. 
+2. Block scale, defined as the union of the geometries of buildings touching each other (at least one point in common). Note that the union can represent a single building. 
+3. Reference Spatial Unit (RSU) scale, being the elementary unit to characterize all the characteristics of a piece of land. The RSU are calculated according to several geographic informations that describe the territory and could have an impact on the urban climate: the road and rail network as well as the vegetation area and water surfaces (when they are sufficiently large to be considered as climatically important – default values are respectively 100’000 m² and 2’500 m²).
```
Comment (j3r3m1) [inline docs/articles/joss/paper.md:87]: should we add a figure for blocks or / and at least RSU ?

File: docs/articles/joss/paper.md (hunk 1 above)
Comment (ebocher) [reply docs/articles/joss/paper.md:84]: where ?

File: docs/articles/joss/paper.md (hunk 2 above)
Comment (ebocher) [reply docs/articles/joss/paper.md:85]: better thanks

File: docs/articles/joss/paper.md (hunk 3 above)
Comment (ebocher) [reply docs/articles/joss/paper.md:87]: nope

**Pull request merged**